File: mcp/tools/serena/test/resources/repos/csharp/test_repo/DiagnosticsSample.cs
using System;

namespace TestProject
{
    public static class DiagnosticsSample
    {
        public static string BrokenFactory()
        {
            return missingGreeting;
        }

        public static void BrokenConsumer()
        {
            string value = BrokenFactory();
            Console.WriteLine(value);
            Console.WriteLine(missingConsumerValue);
        }
    }
}

File: mcp/tools/serena/test/resources/repos/java/test_repo/src/main/java/test_repo/DiagnosticsSample.java
package test_repo;

public class DiagnosticsSample {
    public static String brokenFactory() {
        return missingGreeting;
    }

    public static void brokenConsumer() {
        String value = brokenFactory();
        System.out.println(value);
        System.out.println(missingConsumerValue);
    }
}
